packages/components/src/components/input-field/input.tsx

```diff
@@ -106,13 +107,14 @@ const Input = ({
         <div className={classNameDynamicSuffix}>
             {inline_prefix ? (
                 <div className={classNameInlinePrefix}>
-                    <span
+                    <Text
                         className={classNames(classNameInlinePrefix ? `${classNameInlinePrefix}--symbol` : '', {
                             disabled: !!is_disabled,
                         })}
+                        size='xs'
```

Comment:
I'm replacing some repeatedly used font styles in CSS i found with Text component here as a better practice. QA checklist will include this for all occurences.